docs/events/timeline-ui-overview.asciidoc

```diff
@@ -198,6 +198,8 @@ From the *Correlation* tab, you can also do the following:
 
 preview::["Do not use {esql} on production environments. This functionality is in technical preview and may be changed or removed in a future release. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of official GA features."]
 
+NOTE: The **{esql}** tab is turned on and displays by default. To turn it off and hide it, you must edit your {cloud}/ec-manage-kibana-settings.html#ec-manage-kibana-settings[{kib} user settings] and set the `timelineEsqlTabDisabled` feature flag to `true`. 
```

Comment:
maybe it's just me, but I find it a bit confusing and redundant to say `turned on` and `displays`, and `turn it off` and `hide`. I feel like `turn on` and `turn off` would apply to the feature flag (the configuration/code side) and `displays` and `hide` would apply to the UI side.

Maybe something like this could be better?

```
NOTE: The **{esql}** tab is displayed by default. To hide it, you must edit your {cloud}/ec-manage-kibana-settings.html#ec-manage-kibana-settings[{kib} user settings] and set the `timelineEsqlTabDisabled` feature flag to `true`.
```

Reply:
overall though, the content is correct 😄

Reply:
Hm, I see your point. If we go with "to hide the tab...", do you think there will be any concerns about ES|QL tab processes or queries still running in the background? Would something like the following be more clear and still accurate?

```suggestion
NOTE: The **{esql}** tab is available by default. To deactivate it, you must edit your {cloud}/ec-manage-kibana-settings.html#ec-manage-kibana-settings[{kib} user settings] and set the `timelineEsqlTabDisabled` feature flag to `true`. 
```

Reply:
I feel like nobody would be concerned by the some hidden tab running queries or anything. You and I might wonder that because we know that previously the tab was making queries despite not being rendered, but I wouldn't expect any user to know this.
That being said, `deactivate` works as well (though a bit less clear to me).

Reply:
Gotcha, thanks. Based on your feedback, I have no issues changing "deactivate" to "hide" :) 

```suggestion
NOTE: The **{esql}** tab is available by default. To hide it, you must edit your {cloud}/ec-manage-kibana-settings.html#ec-manage-kibana-settings[{kib} user settings] and set the `timelineEsqlTabDisabled` feature flag to `true`. 
```

Reply:
Agree with Philippe's original point. If it helps, making a suggestion below, but it's probably overkill so feel free to ignore.

NOTE: The **{esql}** tab is available by default. As it is in technical preview, it can be removed by editing your {cloud}/ec-manage-kibana-settings.html#ec-manage-kibana-settings[{kib} user settings] with the `timelineEsqlTabDisabled` setting set to `true`.

Changed two things `hide` => `remove` and `feature flag` => `setting`. The last one is because according to Mike P. our users don't know the concept of feature flags nor does Elastic publicize that term in any way

Reply:
Thanks for the additional feedback here, @michaelolo24! How's this for an alternative version:

NOTE: The {esql} tab is available by default. As it is in technical preview, it can be removed by editing your {cloud}/ec-manage-kibana-settings.html#ec-manage-kibana-settings[{kib} user settings] and setting the `timelineEsqlTabDisabled` feature flag value to `true`.

Reply:
Perfect, thanks!

Reply:
Resolved in [c12bbff](https://github.com/elastic/security-docs/pull/4552/commits/c12bbff0b826fa818d4aa571d7d37026bbd56b51).